lib/magma/commands.rb

```diff
@@ -2,6 +2,50 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    usage '[project_name, path/to/file.json] # Import attributes into database for given project name from JSON file'
+
+    ATTRIBUTES_OPTIONS = [
+      'attribute_name',
```

Comment:
Seems brittle to list these here - could we use `Magma::Attribute.options` instead + a few stragglers in a merge?